backend/battle/tests/test_view.py

```diff
@@ -74,3 +81,178 @@ def test_verify_if_not_returns_winner_missing_pokemons_teams(self):
 
         with self.assertRaises(IndexError):
             get_winner_for(self.team_creator, self.team_opponent)
+
+
+class ListBattlesTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.battle = baker.make('battle.Battle')
+        self.user = User.objects.create(email='example@example.com')
+        self.user.set_password('admin')
+        self.user.save()
+
+    def test_login_user_can_acess_battle_list(self):
+        self.client.login(username=self.user.email, password='admin')
```

Comment:
Since this is being used on a lot of classes, you should extract this to a separate class and create an `auth_client`

Reply:
Yup! Thanks :)